lisa/tools/mkfsext.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mkfsext(Tool):
+    __EXIST_FILE_SYSTEM_PATTERN = re.compile(
+        r".*appears to contain an existing filesystem", re.MULTILINE
+    )
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def mkfs(self, disk: str, command: str = "mkfs.ext4") -> None:
```

Comment:
add doc string to introduce what the method do, what can be filled into the command? Why the command need to be an argument?